source/app/documentmanager.h

```diff
@@ -71,7 +72,12 @@ class DocumentManager
 
     bool hasOpenDocuments() const;
     void setCurrentDocumentIndex(int index);
+
     int getCurrentDocumentIndex() const;
+	size_t getDocumentListSize() const;
+	
+	/// Returns -1 if the file at filepath is not open, else it returns the index at which the already open file is at
+	int containsDocument(const std::string& filepath);
```

Comment:
It might make more sense to rename the method to `findDocument`, since I would expect `containsDocument` to return a bool.